gtsam/basis/basis.i

```diff
@@ -64,7 +68,7 @@ class ParameterMatrix {
 #include <gtsam/basis/BasisFactors.h>
 
 template <BASIS = {gtsam::Chebyshev2}>
-class EvaluationFactor : gtsam::NoiseModelFactor {
+virtual class EvaluationFactor : gtsam::NoiseModelFactor {
```

Comment:
Does this have to be `virtual`?

Reply:
Just for future reference, ya I was getting a compile (or linker?) error without it:
https://github.com/borglab/wrap/blob/master/DOCS.md
> * Virtual inheritance
>   * ...
>   * Mark with virtual keyword, e.g. virtual class Base {, and also virtual class Derived : ns::Base {.

:)

Reply:
Yeah I am aware of the docs, but the pybind wrapper doesn't quite use virtual anywhere for subclasses. I may be wrong, but the virtual is supposed to be mainly for base classes.

Gotta look it up and update docs.